x-pack/plugins/security_solution/public/timelines/components/timeline/body/stateful_body.tsx

```diff
@@ -197,6 +201,7 @@ const StatefulBodyComponent = React.memo<StatefulBodyComponentProps>(
         pinnedEventIds={pinnedEventIds}
         rowRenderers={showRowRenderers ? rowRenderers : [plainRowRenderer]}
         selectedEventIds={selectedEventIds}
+        show={id === ACTIVE_TIMELINE_REDUX_ID ? show : true}
```

Comment:
I will prefer to use TimelineId enum here

Reply:
I will add this in the follow-up PR